### PR TITLE
Adding location for Vedika OH

### PR DESCRIPTION
**Pull request by VedikaSrivastava**

### CODE REVIEW

File: _staffers/VedikaSrivastava [hunk 1]
```diff
@@ -6,4 +6,4 @@ lab: B4 (Thu 5:00pm-5:50pm)
 photo: Vedika_Srivastava.jpg
 ---
 
-Office Hours: 4pm - 5pm Location: TBD
+Office Hours: T 4:00pm - 5:00pm Location: CDS 801
```
Comment (gallettilance) [inline _staffers/VedikaSrivastava:9]: T = Tuesday right?

Comment (VedikaSrivastava) [reply _staffers/VedikaSrivastava:9]: Yess

Comment (VedikaSrivastava) [reply _staffers/VedikaSrivastava:9]: should i write it as Tue?

**Comment by VedikaSrivastava** [comment]
I have update the file to show "Tue" instead of "T"

**Pull request merged**